Makefile

```diff
@@ -60,3 +60,10 @@ test-unit:
 .PHONY: test-integration
 test-integration:
 	@if [[ "$(tableName)" == "" ]]; then go test -run=TestIntegration -timeout 3m -tags=integration ./...; else go test -run="TestIntegration/$(tableName)" -timeout 3m -tags=integration ./...; fi
+
+# Create a DB migration
+.PHONY: db-migration
+db-migration:
+	@if [[ "$(prefix)" == "" ]]; then echo "Invalid prefix, see example 'make db-migration prefix=9_v0.9.0'" && exit 1; fi;
+	go run tools/migrations/main.go -prefix "${prefix}" -dsn 'postgres://postgres:pass@localhost:5432/postgres?sslmode=disable'
+	go run tools/migrations/main.go -prefix "${prefix}" -fake-tsdb -dsn 'postgres://postgres:pass@localhost:5432/postgres?sslmode=disable'
```

Comment:
should be `tsdb://` I think

Reply:
Shouldn't the `-fake-tsdb` flag take care of that (Looking at the code [here](https://github.com/cloudquery/cq-provider-sdk/blob/9228d7304d193dd249cc47ff9bda9f683629068d/migration/cli.go#L75))?

Happy to change it though. Should it be `tsdb://` to `timescale://`? Our docs say `timescale://` https://docs.cloudquery.io/docs/developers/sdk/migrations/migration-tool#dialect-url-schemes so I'm happy to follow up on those too